Makefile

```diff
@@ -19,7 +19,7 @@ tutorial: FORCE
 
 lint: FORCE
 	flake8
-	black --extend-exclude=\.ipynb --check .
+	black --extend-exclude=\.ipynb --check . || black --exclude=\.ipynb --check .
```

Comment:
ditto, it's better to pin black version

Reply:
That's a good point! I didn't think about what happens when there are errors.